File: 0283-move-zeroes/0283-move-zeroes.java
class Solution {
    public void moveZeroes(int[] nums) {
        int s=0;
        for(int f=0;f<nums.length;f++){
            
            if(nums[f]!=0){
                if(s != f){
                    nums[s] = nums[f];
                    nums[f] = 0;
                }
                s++;
            }
        }
    }
}

File: 0283-move-zeroes/0283-move-zeroes.py
class Solution:
    def moveZeroes(self, nums: List[int]) -> None:
        s=0
        for f in range(0,len(nums)):
            if nums[f]!=0:
                nums[s],nums[f]=nums[f],nums[s]
                s+=1
        return nums
        